<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1026</width>
    <height>776</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1026</width>
    <height>776</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Цифровой приемный тракт РПУ &quot;Кальмар&quot;</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/buttons/SquidLow.png</normaloff>:/buttons/SquidLow.png</iconset>
  </property>
  <widget class="QWidget" name="widgetInterface" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>762</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>762</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>762</height>
    </size>
   </property>
   <widget class="QProgressBar" name="ReadProgressBar">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>30</y>
      <width>928</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="WindowTitleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>1039</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="RecButton">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ConfigButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>26</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>996</x>
      <y>3</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="MinimizeButton">
    <property name="geometry">
     <rect>
      <x>972</x>
      <y>3</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="simpleGraph" name="FftGraph" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>51</y>
      <width>1024</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>711</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>711</height>
     </size>
    </property>
    <widget class="QSlider" name="DynamicRangeSlider">
     <property name="geometry">
      <rect>
       <x>1002</x>
       <y>6</y>
       <width>20</width>
       <height>371</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSlider" name="NoiseLevelSlider">
     <property name="geometry">
      <rect>
       <x>1002</x>
       <y>380</y>
       <width>20</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-10</number>
     </property>
    </widget>
   </widget>
   <zorder>WindowTitleLabel</zorder>
   <zorder>ReadProgressBar</zorder>
   <zorder>RecButton</zorder>
   <zorder>StopButton</zorder>
   <zorder>ConfigButton</zorder>
   <zorder>CloseButton</zorder>
   <zorder>MinimizeButton</zorder>
   <zorder>FftGraph</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>simpleGraph</class>
   <extends>QWidget</extends>
   <header location="global">simplegraph.h</header>
   <container>1</container>
   <slots>
    <slot>setNoiseLevel(int)</slot>
    <slot>setDynamicRange(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
